lib/src/CoE/mailbox/response.cc

```diff
@@ -301,6 +303,239 @@ namespace kickcat::mailbox::response
         }
     }
 
+
+    SDOInformationMessage::SDOInformationMessage(Mailbox* mbx, std::vector<uint8_t>&& raw_message)
+        : AbstractMessage{mbx}
+    {
+        data_ = std::move(raw_message);
+
+        header_  = pointData<mailbox::Header>(data_.data());
+        coe_     = pointData<CoE::Header>(header_);
+        sdo_     = pointData<CoE::ServiceDataInfo>(coe_);
+    }
+
+
+    ProcessingResult SDOInformationMessage::process()
+    {
+        /*
```

Comment:
debug ?

Reply:
It was more somethink like "I probably need to check for ill-formed message" but at the end I cannot find information about that nor no tests in the CTT. I'll clean it